std/machines/write_once_memory_with_8_publics.asm

```diff
@@ -0,0 +1,32 @@
+// Very simple write-once memory that let's you store any field element at
```

Comment:
```suggestion
// Very simple write-once memory that lets you store any field element at
```

Reply:
updated, also in write_once_memory which was the base for this file